src/coremidi.cs

```diff
@@ -228,5 +253,172 @@ interface MidiCISession
 
 		[NullAllowed, Export ("propertyChangedCallback", ArgumentSemantic.Assign)]
 		MidiCIPropertyChangedHandler PropertyChangedCallback { get; set; }
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("initWithDiscoveredNode:dataReadyHandler:disconnectHandler:")]
+		IntPtr Constructor (MidiCIDiscoveredNode discoveredNode, Action handler, MidiCISessionDisconnectHandler disconnectHandler);
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("sendProfile:onChannel:profileData:")]
+		bool SendProfile (MidiCIProfile profile, byte channel, NSData profileSpecificData);
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("deviceInfo")]
+		MidiCIDeviceInfo DeviceInfo { get; }
+
+		[Mac (10, 16), iOS (14, 0)]
+		[Export ("maxSysExSize")]
+		NSNumber MaxSysExSize { get; }
```

Comment:
do we know the underlying type ? i.e. `[BindAs]`

Reply:
Not sure about the headers but the [docs](https://developer.apple.com/documentation/coremidi/midicisession/3580337-maxsysexsize) are not very helpful unfortunately 😞 maybe have it now as `NSNumber` and add it to the list https://github.com/xamarin/xamarin-macios/issues/8943 to check at the end of the summer? What do you think @spouliot ?

Reply:
The MIDI[-CI] specs are more useful